boards/same54-xpro/include/periph_conf.h

```diff
@@ -148,6 +148,35 @@ static const uart_conf_t uart_config[] = {
 #define UART_NUMOF          ARRAY_SIZE(uart_config)
 /** @} */
 
+/**
+ * @name PWM configuration
+ * @{
+ */
+#define PWM_0_EN            1
+
+#if PWM_0_EN
+/* PWM0 channels */
+static const pwm_conf_chan_t pwm_chan0_config[] = {
+    /* GPIO pin, MUX value, TCC channel */
+    { GPIO_PIN(PC, 18), GPIO_MUX_F, 2 },
```

Comment:
Why choosing this pin ? It is not available on expander connectors.
PC18 is only available through DGI or PDEC on SAME54.

Reply:
It's the on-board LED - so you can get immediate feedback with the `osci` command.